src/app/feature/consultant/components/consultant-experience/consultant-experience-form/consultant-experience-form.component.ts

```diff
@@ -106,14 +108,20 @@ export class ConsultantExperienceFormComponent implements OnInit, OnDestroy {
 
       experience.push(newExperience);
       this.close(experience);
+      this.notification.openUpdatedSnackBar();
+
     }
   }
 
   private updateExperience(): void {
+    const experience = this.consultantStore.consultant.experience;
```

Comment:
Please revert these changes and move the snackbar message handling into the parent ConsultantExperienceComponent. This component doesn't actually do any updating, it just passes the updated data to the parent which then handles the updating and that's where updating could fail.